docs/cookbook/recipe_custom_view.rst

```diff
@@ -0,0 +1,134 @@
+Creating a Custom Admin View
+============================
+
+Sometimes you may want to create an additional view in your SonataAdmin
+but you do not have or need an Entity to connect it to. You also probably
+want to add it to sidebar with the rest of your admin pages? Here we will
+present a full working example of how is that achievable.
+
+The recipe
+----------
+
+SonataAdmin provides a very straightforward way of adding your own custom
+view.
+
+To do this we need to:
+
+- create an admin class for your custom view
+- extend the ``SonataAdmin:CRUD`` Controller and tell our admin class to
+  use it
+- create a template that will represent your custom view
+
+Create an admin class
+^^^^^^^^^^^^^^^^^^^^^
+
+First we need to create an admin class and we will clear all routes except
+``list``.
+
+.. code-block:: php
+
+    <?php
+
+    // src/Admin/CustomViewAdmin.php
+
+    namespace App\Admin;
+
+    use Sonata\AdminBundle\Admin\AbstractAdmin;
+    use Sonata\AdminBundle\Route\RouteCollection;
+
+    class CustomViewAdmin extends AbstractAdmin
+    {
+        protected $baseRoutePattern = 'custom_view';
+        protected $baseRouteName = 'custom_view';
+
+        protected function configureRoutes(RouteCollection $collection)
+        {
+            $collection->clearExcept(['list']);
+        }
+    }
+
+.. note::
```

Comment:
I do not know if this note is necessary really, this namespace depends on the application. Im +1 on using how flex comes by default. But if you are not using flex, I guess you know how to change this to adapt to your bundle namings, right? wdyt?

Reply:
I agree, removed.